README.md

```diff
@@ -135,23 +135,33 @@ There is also a [tapioca-wrapper](https://github.com/vintasoftware/tapioca-wrapp
 If you have [Docker](https://docs.docker.com/engine/installation/) (with [Docker Compose](https://docs.docker.com/compose/install/)) and make, just run:
 
 ```console
-$ docker-compose up -d --build
-$ docker-compose run --rm jarbas python manage.py migrate
-$ docker-compose run --rm jarbas python manage.py ceapdatasets
+make run.devel
 ```
 
-You can access it at [`localhost:80`](http://localhost:80/). However your database starts empty and you still have to collect your static files:
+or
 
 ```console
-$ docker-compose run --rm jarbas python manage.py collectstatic --no-input
-$ docker-compose run --rm jarbas python manage.py reimbursements <path to reimbursements.xz>
-$ docker-compose run --rm jarbas python manage.py irregularities <path to irregularities.xz file>
-$ docker-compose run --rm jarbas python manage.py companies <path to companies.xz>
+docker-compose up -d --build
+docker-compose run --rm jarbas python manage.py migrate
+docker-compose run --rm jarbas python manage.py ceapdatasets
+docker-compose run --rm jarbas python manage.py collectstatic --no-input
 ```
 
-You can get the datasets running [Rosie](https://github.com/datasciencebr/rosie) or directly with the [toolbox](https://github.com/datasciencebr/rosie).
+You can access it at [`localhost:80`](http://localhost:8080/). However your database starts empty, but you can use sample data to development using this command:
```

Comment:
Text says port `80`, but the link and `docker-compose.yml` says `8080`. For _dev_ I'd opt for `8000` (Django's default).